web/lens/src/styles.css

```
@@ -241,6 +273,169 @@
+  .lens-table-pagination button:disabled {
+    @apply lens-opacity-40;
+  }
+
+  .lens-sr-only {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)